config/webpack/development.js

```diff
@@ -0,0 +1,3 @@
+const environment = require('./environment')
+
+module.exports = environment.toWebpackConfig()
```

Comment:
Typically, you'd set up `webpack` using a config file [like this](https://webpack.js.org/configuration/).

Webpacker apparently takes care of this for us - all of the relevant config options are now in `config/webpacker.yml`, and they get translated into JS in this file.

Reply:
If we want to customize it further, we can override options here, or in the other `config/webpack/*.js` files.